.github/workflows/labeler.yml

```diff
@@ -0,0 +1,16 @@
+name: Pull Request Labeler
+on: pull_request_target
+# By default, pull_request_target runs when the activity type is opened, synchronize, or reopened
+# https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#pull_request_target
+
+jobs:
+  categorise:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/labeler@v4
+      with:
+        configuration-path: '.github/project-labeler.yml'
+        repo-token: '${{ secrets.GITHUB_TOKEN }}'
```

Comment:
is the token defined in the repo?

Reply:
Yep, it's a default token provided by Github to allow actions to take actions on the repo, such as automatic PR's / labelling etc.